test/e2e/slack_driver_test.go

```diff
@@ -181,47 +181,8 @@ func (s *slackTester) WaitForMessagePosted(userID, channelID string, limitMessag
 }
 
 func (s *slackTester) WaitForInteractiveMessagePosted(userID, channelID string, limitMessages int, assertFn MessageAssertion) error {
```

Comment:
If this is no longer used, we should delete the method and remove it from the BotDriver interface.

Reply:
Added implementation, like you did in Discord